app/modules/users/resources.py

```diff
@@ -276,3 +276,27 @@ def get(self, refresh=False):
         }
 
         return response
+
+
+@api.route('/<uuid:user_guid>/sightings')
+@api.resolve_object_by_model(User, 'user')
+class UserSightings(Resource):
+    """
+    EDM Sightings for a given Houston user
+    """
+
+    @api.login_required(oauth_scopes=['users:read'])
```

Comment:
Why does this not use the ObjectAccessPermission? 
This would allow any user to read any other users sightings as it is. That must be wrong surely?